sky/sdk/example/stocks/lib/stock_home.dart

```diff
@@ -252,7 +252,12 @@ class StockHome extends StatefulComponent {
       showing: _isSnackBarShowing,
       content: new Text("Stock purchased!"),
       actions: [new SnackBarAction(label: "UNDO", onPressed: _handleUndo)],
-      onStatusChanged: (status) { setState(() { _snackBarStatus = status; }); }
+      onStatusChanged: (status) {
+        // TODO(mpcomplete): should the snackbar call this in a microtask?
+        scheduleMicrotask(() {
```

Comment:
Why do we need this?  It definitely seems like something the snackbar should do itself rather than having every client have this mysterious code.